snapheka/snapheka.go

```diff
@@ -34,14 +34,16 @@ import (
 )
 
 const (
-	name       = "heka"
-	version    = 2
+	//vendor namespace part
+	vendor     = "intel"
+	pluginName = "heka"
+	version    = 3
 	pluginType = plugin.PublisherPluginType
 )
 
 // Meta returns a plugin meta data
 func Meta() *plugin.PluginMeta {
-	return plugin.NewPluginMeta(name, version, pluginType, []string{plugin.SnapGOBContentType}, []string{plugin.SnapGOBContentType})
+	return plugin.NewPluginMeta(pluginName, version, pluginType, []string{plugin.SnapGOBContentType}, []string{plugin.SnapGOBContentType})
```

Comment:
The option plugin.RoutingStrategy(plugin.StickyRouting) should be added to ensure that there is a 1:1 relationship between running tasks and running plugin instances.